themes/default/content/docs/pulumi-cloud/self-hosted/components/api.md

```diff
@@ -288,3 +288,150 @@ To use [AWS DynamoDB](https://aws.amazon.com/dynamodb) to persist Audit Logs, sp
       },
   ],
 ```
+
+## OpenTelemetry
+
+The API service is configured to export OpenTelemetry metrics and traces to the vendor of your choice via the OpenTelemetry collector. You will need to manage your own OpenTelemetry collector.
+
+The following environment variables are needed to configure OpenTelemetry in the service:
+
+| Variable name | Description |
+| --- | --- |
+| OTEL_EXPORTER_OTLP_ENDPOINT | (Required) Used to configure the OTLP exporter. The base URL to which all telemetry will be sent. If not set, the API service will use no-op metrics and traces. |
+| OTEL_EXPORTER_OTLP_PROTOCOL | (Optional) Used to configure the OTLP exporter. Valid values are `http` or `grpc`. Defaults to `grpc`. |
+| PULUMI_ENABLE_DEPRECATED_METRICS | (Optional) Whether to continue emitting API service metrics in a log-based format. Defaults to `true`. |
+| METRICS_WEBHOOK_SECRET | Required to successfully authenticate to the `/metrics` endpoint. The Authorization header shoud be set as follows: `Authorization: webhook-token <METRICS_WEBHOOK_SECRET>`. |
+
+OpenTelemetry is not yet available for the node backend or console.
```

Comment:
These are the same thing in terms of services, right? (Apologies if not -- it's been a while.) If so, should we just call this "the console service" (as that's what it seems to be called in the sibling docs)?

```suggestion
OpenTelemetry is not yet available for the [console service](/docs/pulumi-cloud/self-hosted/components/console/).
```